FreshRSS/FreshRSS
# fix broken sharing services

Closes #3793 

Changes proposed in this pull request:

- Blogotext: deleted
- Lemmy: set link
- Movim: changed link
- mastadon: set link
- reordered array

How to test the feature manually:
1. go to integration/share settings
2. edit/add the services
3. check the help buttons

Pull request checklist:
